biscuit-auth/src/token/mod.rs

```diff
@@ -221,6 +221,19 @@ impl Biscuit {
         })
     }
 
+    /// prints the content of a block as a protobuf message
+    pub fn print_fact(&self, fact: &crate::datalog::Fact) -> String {
+        self.symbols.print_fact(fact)
+    }
+
+    pub fn print_rule(&self, rule: &crate::datalog::Rule) -> String {
+        self.symbols.print_rule(rule)
+    }
+
+    pub fn print_check(&self, check: &crate::datalog::Check) -> String {
+        self.symbols.print_check(check)
+    }
```

Comment:
I don't know if it makes sense, maybe there is a better way

Reply:
I think we should remove the `biscuit_block_fact`, `biscuit_block_rule` and `biscuit_block_check` functions from the C API. There is no equivalent in the rust API, because it does not make much sense to print facts or rules individually by their index. Instead there is `print_block_source` to show the datalog of an entire block

Reply:
done here: [`00dfed0` (#234)](https://github.com/biscuit-auth/biscuit-rust/pull/234/commits/00dfed05c8cf2af9cb69052ba9b6436ff2169758)

Thanks for the suggest!